blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionHandler.java

```diff
@@ -81,6 +81,9 @@ protected JSONMetricsContainer createContainer(String body, String tenantId) thr
     @Override
     public void handle(ChannelHandlerContext ctx, HttpRequest request) {
         try {
+
+            Tracker.track(request);
```

Comment:
@tilogaat so at the moment, this can not be easily done because HttpRequestHandler is an inteface and not a base class.  What we would have to do is to convert HttpRequestHandler to an abstract class, or create a base class (which can also be abstract) that implements HttpRequest, then modify all the current Handlers to inherits from the abstract class instead.  

I am not sure if we want that amount of refactoring done for this story, but if you think we should, I can do it.  Or is there a different solution you have in mind that we can do to ensure Tracker.track is called for handle()?